src/applications/personalization/appointments/reducers/index.js

```diff
@@ -2,7 +2,7 @@ import {
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS,
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS_SUCCEEDED,
-} from '~/applications/personalization/dashboard/constants';
+} from '../actions/index';
```

Comment:
Nit:
this can just be '../actions' as the index is implied and module will be resolved without explicitly using the index name in the path.